### ISSUE TITLE
npc-duplicate remove server crash

### ISSUE THREAD

**Issue by Mojacko** [dastgirp/HPM-Plugins#46]
Whenever i try to use the duplicateremove function the map-server crash
![image](https://user-images.githubusercontent.com/7709489/74004977-63777180-49b2-11ea-8a17-94bb65a3bc3d.png)

the GDB says at script.c:3838 
![image](https://user-images.githubusercontent.com/7709489/74005018-830e9a00-49b2-11ea-92de-861418674286.png)

**Comment by Mojacko** [comment]
Update: I tried using emistry pr and it seems to be working fine
https://github.com/HerculesWS/Hercules/pull/2473/commits/db6fe143dfe5e5d95646f741e1b8bd53fa9709fa

**Comment by dastgirp** [comment]
Yes, as that is added to source, I will remove it from here and add documentation
